crates/nu-command/tests/commands/random/chars.rs

```diff
@@ -3,7 +3,7 @@ use nu_test_support::nu;
 #[test]
 fn generates_chars_of_specified_length() {
     let actual = nu!(r#"
-        random chars --length 15 | size | get chars
+        random chars --length 15 | str size | get chars
```

Comment:
```suggestion
        random chars --length 15 | str stats | get chars
```

Reply:
because `size` is still a thing in this PR and this change is already done in https://github.com/nushell/nushell/pull/10784/files#diff-30aa1b02642f5be3bd8b4a6d2391d13c86bb8125c6a4085a23ea97c169e9fa5a, i think we can leave that as it is :relieved: